release/ray_release/byod/byod_gptj_test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
need to make this file executable it seems

Reply:
yep; fixed.